src/v2/coins/abstractUtxoCoin.js

```diff
@@ -548,7 +548,7 @@ class AbstractUtxoCoin extends BaseCoin {
     }
 
     if (_.isUndefined(addressType)) {
-      addressType = Codes.typeForCode(derivationChain);
+      addressType = Codes.UnspentTypeTcomb('p2sh');
```

Comment:
This is a change in behavior if `chain` is not zero. This change should rather be applied to line 545.

Reply:
If you mean from line 545 of the file before these changes then that is where it is applied. I just switched the order of the `if` statement such that we don't need to have an `else if` anymore.

Reply:
If i'm missing something, please elaborate, but previously, when `chain` was `> 0`, `addressType` would be set in line with `derivationChain`. Now it is always set to p2sh. This seems to be a breaking change.

Reply:
Nevermind, I was looking at the commits one by one, but this was changed and reverted in different commits.